.azure/pipelines/ci.yml

```diff
@@ -17,383 +17,401 @@ pr:
     include:
     - '*'
 
-jobs:
-- template: jobs/default-build.yml
-  parameters:
-    jobName: Code_check
-    jobDisplayName: Code check
-    agentOs: Windows
-    steps:
-    - powershell: ./eng/scripts/CodeCheck.ps1 -ci
-      displayName: Run eng/scripts/CodeCheck.ps1
+stages:
+- stage: Build
+  displayName: ' '
```

Comment:
Something in AzDO uses this prefix for every job in the main stage when the "Multi-stage pipelines" preview feature is off. Making the display name non-`null` and non-empty avoids that ugly repetition.

Reply:
On the other hand, this now causes the new UI to display nothing for the stage name. I think we should encourage the new UI and avoid workarounds like this for the old UI...which is going away relatively soon.

![image](https://user-images.githubusercontent.com/2696087/58042256-07508280-7aef-11e9-9449-589d35aceba5.png)